pl_examples/domain_templates/semantic_segmentation.py

```diff
@@ -17,6 +17,29 @@
 DEFAULT_VALID_LABELS = (7, 8, 11, 12, 13, 17, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 31, 32, 33)
 
 
+def _create_synth_Cityscapes_dataset(path_dir):
```

Comment:
Why do we need this? This isn't used?

Reply:
it is til wip, and the point here is to create a fake dataset if two images and skip downloading full dataset...